### PR TITLE
hull-node upgraded to 0.11.0@beta.3 version

### PR DESCRIPTION
**Pull request by mkawalek**

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -1,7 +1,6 @@
-FACEBOOK_APP_ID=
-FACEBOOK_APP_SECRET=
+FACEBOOK_APP_ID=563491867154892
```
Comment (michaloo) [inline .env.sample]: Ooops, watch out for commiting the credentials here. This is some testing account? Let's refresh it.

File: server/facebook-audience.js [hunk 2]
```diff
@@ -90,7 +90,7 @@ export default class FacebookAudience {
     }
 
     return BatchSyncHandler.getHandler({
-      hull, ship,
+      client, ship,
```
Comment (michaloo) [inline server/facebook-audience.js]: Here you pass client, but the `BatchSyncHandler` expects `hull` here. Can you verify that?

File: server/facebook-audience.js [hunk 3]
```diff
@@ -192,18 +203,16 @@ export default class FacebookAudience {
   }
 
   createAudience(segment, extract = true) {
-    this.metric("ship.audience.create");
+    this.incMetric("ship.audience.create");
```
Comment (michaloo) [inline server/facebook-audience.js]: You could replace `this.incMetric` with `this.metric.inc` and drop that helper method.

File: server/server.js [hunk 4]
```diff
@@ -17,31 +13,17 @@ function onError(err, req, res, next) { // eslint-disable-line no-unused-vars
...
-  const { BatchHandler, NotifHandler, Routes } = Hull;
-
-  const app = express();
-  app.engine("html", ejs.renderFile);
-  app.set("views", path.resolve(__dirname, "..", "views"));
-
+module.exports = function Server(options) {
+  const { connector, app, facebookAppId, facebookAppSecret, sentryDSN } = options;
 
   if (sentryDSN) {
```
Comment (michaloo) [inline server/server.js]: you can remove that part - the sentry instrumentation is handled by the `Hull.Connector`.

File: server/server.js [hunk 5]
```diff
@@ -52,28 +34,28 @@ module.exports = function Server({ Hull, port, facebookAppId, facebookAppSecret,
...
+      fb.addUsersToAudience(audience, users);
     }
+  }, {
+    groupTraits: false
   }));
 
-  app.use("/admin", adminHandler({ Hull, facebookAppSecret, facebookAppId }));
+  app.use("/admin", adminHandler({ connector, facebookAppSecret, facebookAppId }));
 
   if (sentryDSN) {
```
Comment (michaloo) [inline server/server.js]: to remove.

File: .env.sample (hunk 1 above)
Comment (mkawalek) [inline .env.sample]: Yes, it was testing account so no harm done :)

File: server/facebook-audience.js (hunk 2 above)
Comment (mkawalek) [inline server/facebook-audience.js]: Actually, BatchSyncHandler was not using this hull object so I removed it. It looks cleaner now :)

**Comment by michaloo** [comment]
[hull-facebook-audiences](https://trello.com/c/leg893OH/13-hull-facebook-audiences)

**Comment by michaloo** [comment]
Since develop is autodeploying here, I'm replacing this PR with local branch.

**Pull request closed without merging**